Don't fail if unrecognised command-line flags are present.
Permit building if security/openpam is installed.

Index: configure
--- configure.orig
+++ configure
@@ -2825,11 +2825,6 @@ ac_config_headers="$ac_config_headers config.h"
 echo "current directory: `pwd`"
 echo "command line was: $0 $@"
 
-if ! test -z "$ac_unrecognized_opts" ; then
-  echo "" >&2
-  exit 2
-fi
-
 ###############################################################################
 #
 #       Autoheader stuff
@@ -15042,9 +15037,9 @@ fi
 
 # Check for the various Gnome help and URL loading programs.
 #
-WITH_BROWSER=gnome-open
+WITH_BROWSER=xdg-open
 if test "$have_gtk" = yes; then
-  for ac_prog in gnome-open
+  for ac_prog in xdg-open
 do
   # Extract the first word of "$ac_prog", so it can be a program name with args.
 set dummy $ac_prog; ac_word=$2
@@ -22069,7 +22064,6 @@ elif test "$have_pam" = no ; then
   if test -d /etc/pam.d -o -f /etc/pam.conf ; then
     warn  "Your system seems to have PAM, but PAM is not being used."
     warn2 "That is probably not going to work out well."
-    CONF_STATUS=1
   fi
 fi
 
